core/rpc/src/system/helpers.rs

```diff
@@ -51,7 +51,7 @@ pub struct Health {
 }
 
 /// Network Peer information
-#[derive(Debug, PartialEq, Serialize)]
+#[derive(Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
Why do they now need deserialize?

Reply:
Otherwise, the code did not compile, IIRC.

Reply:
Could you please post the error?